ado/ipatrackversions.ado

```diff
@@ -0,0 +1,195 @@
+*! version 1.0.0 Caton brewster 10nov2016
+
+program ipatrackversions, rclass
+	/* Create a sheet that shows the survey versions used for each day of surveying. 
+	For the most recent day of surveying, if there are surveys that have been submitted
+	using the wrong version, list the enumerator ID, respondent ID, etc. for those observations
+	to facilitate finding that enumerator and ensuring they upload the latest version of the 
+	survey */
+	
+	
+
+//define inputs
+version 13
+
+#delimit ;
+syntax varname,  //varname is the form version variable - must be num. 
+	/* specify uid for the survey, enumerator id */
+    id(varname) ENUMerator(varname) 
+	/* list of other vars to keep */
+	[KEEPvars(string)] 
+	/* specify date var, i.e. submission date var */
+	submit(varname numeric)
+	/* output filename */
+	saving(string) 
+	;	
+#delimit cr
+
+	
+	di ""
+	di "Compiling information on survey form versions by submission date..."
+
+	qui {
+	
+	* test for fatal conditions 
+	cap confirm numeric var `varlist'
+	if _rc {
+		di as err "Variable used for form version (`varlist') not numeric. Must be numeric and ordinal for the program to work correctly"
```

Comment:
Oh duh. Sorry I misread this part and thought you were checking that the submission date variable was numeric.